### DISCUSSION TITLE
npm ERR! cb() never called!

### DISCUSSION THREAD

**Discussion by sarowarhosen03** [npm/feedback#730] [CLI] [answered]
Am seeing this error when try to update npm version via this command npm install -g npm
![Screenshot_20220716-233649](https://user-images.githubusercontent.com/98280301/179366215-cfdea7c0-e9aa-4d29-a958-8388a7ed9c57.jpg)

**Comment by MylesBorins** [comment, marked as answer]
This forum is intended for suggesting improvements to the npmjs website, registry, and CLI... not general technical support.

It also looks like you are running on android, which is not an officially supported platform.

I'd advise updating the version of Node.js to update your version of npm

**Reply by sarowarhosen03** [reply]
> This forum is intended for suggesting improvements to the npmjs website, registry, and CLI... not general technical support.
> 
> It also looks like you are running on android, which is not an officially supported platform.
> 
> I'd advise updating the version of Node.js to update your version of npm
Yeah you are right 😍
